api-references/payments/bbps.json

```diff
@@ -2575,7 +2575,8 @@
                     "required": [
                         "amountPaid",
                         "billAmount",
-                        "platformTransactionRefID"
+                        "platformTransactionRefID",
```

Comment:
This parameter has been added as required but I don't see it in the properties array below. As a result, it's not getting rendered in the request body (SS for reference). 

<img width="472" alt="Screenshot 2023-12-14 at 8 02 42 PM" src="https://github.com/SetuHQ/docs/assets/9695866/082c03a9-7255-4164-8060-3f5eb67f2782">